pkgs/top-level/python-packages.nix

```diff
@@ -661,6 +661,8 @@ in {
     };
   };
 
+  autoflake = callPackage ../development/python-modules/autoflake { };
+
   autopep8 = callPackage ../development/python-modules/autopep8 { };
```

Comment:
`autopep8` might be also a candidate... but this does not need to be done in this pr.